orcid_hub/forms.py

```diff
@@ -257,6 +257,73 @@ class FundingForm(FlaskForm):
     disambiguation_source = StringField("Disambiguation Source")
 
 
+class PeerReviewForm(FlaskForm):
+    """User/researcher Peer review detail form."""
+
+    reviewer_role_choices = [('MEMBER', 'MEMBER'), ('REVIEWER', 'REVIEWER'), ('ORGANIZER', 'ORGANIZER'),
```

Comment:
so many duplicates!

This might do better:

```python

[(v, v) for v in ['MEMBER', 'REVIEW', ...]]

```
However, it might be that the choice list is a list of atomic values instead of tuples, the WTForms would use the value both for the item value and display value.